pkg/features/versioned_kube_features.go

```diff
@@ -690,7 +690,7 @@ var defaultVersionedKubernetesFeatureGates = map[featuregate.Feature]featuregate
 	},
 
 	ServiceAccountNodeAudienceRestriction: {
-		{Version: version.MustParse("1.32"), Default: true, PreRelease: featuregate.Beta},
```

Comment:
also open a master branch PR that toggles this to `false` for 1.32 and records 1.33 as the starting point for `true`